apps/condo/domains/analytics/components/Dashboard.tsx

```diff
@@ -94,7 +94,6 @@ const PerformanceCard = ({ organizationId, paymentSum, receiptSum, residentsData
         onCompleted: (result) => {
             ticketCounts.current = result
         },
-        fetchPolicy: 'cache-first',
```

Comment:
`cache-first` is default value :)

[Setting a fetch policy](https://www.apollographql.com/docs/react/data/queries/#setting-a-fetch-policy)